Feature/ckeditor4 test (#925)
plentymarkets/terra-components: CHANGELOG.md

> <a name="4.0.0-beta.X"></a>

Reviewer: ```suggestion
<a name="4.0.0-beta.4"></a>
```

> # 4.0.0-beta.X (Unreleased)

Reviewer: ```suggestion
# 4.0.0-beta.4 (Unreleased)
```

### Features
* **tcCkFull** New directive to provide a full config for the ckeditor4.

> * **ck-editor-presets** Preset configs for the ckeditor4.

Reviewer: ```suggestion
* **ck-editor-presets** Config presets for the ckeditor4.
```
